sources/packages/backend/libs/integrations/src/services/ssh/sftp-integration-base.ts

```diff
@@ -81,7 +81,13 @@ export abstract class SFTPIntegrationBase<DownloadType> {
 
     return filesToProcess
       .map((file) => path.join(remoteDownloadFolder, file.name))
-      .sort();
+      .sort((file1: string, file2: string) => {
```

Comment:
It can be simplified and still be Sonarcloud compatible, IMO.
```ts
.map((file) => path.join(remoteDownloadFolder, file.name))
.sort((a, b) => a.localeCompare(b));
```